src/abi_ppc64le.cpp

```diff
@@ -0,0 +1,111 @@
+//===-- abi_ppc64le.cpp - Power v2 ABI description ---------------------*- C++ -*-===//
+//
+//                         LDC – the LLVM D compiler
+//
+// This file is distributed under the BSD-style LDC license:
```

Comment:
I'll fix this one. There's a bunch more that need to be fixed though (`src/support/dirname.c` is missing from the list and the script needs to be rerun).